blocks/library/cover-image/index.js

```diff
@@ -130,21 +130,24 @@ export const settings = {
 				/>
 
 				{ alignmentToolbar }
-				<Toolbar>
-					<MediaUpload
-						onSelect={ onSelectImage }
-						type="image"
-						value={ id }
-						render={ ( { open } ) => (
-							<IconButton
-								className="components-toolbar__control"
-								label={ __( 'Edit image' ) }
-								icon="edit"
-								onClick={ open }
-							/>
-						) }
-					/>
-				</Toolbar>
+
+				{ url ?
+					<Toolbar>
+						<MediaUpload
+							onSelect={ onSelectImage }
+							type="image"
+							value={ id }
+							render={ ( { open } ) => (
+								<IconButton
+									className="components-toolbar__control"
+									label={ __( 'Edit image' ) }
+									icon="edit"
+									onClick={ open }
+								/>
+							) }
+						/>
+					</Toolbar> :
+					'' }
```

Comment:
It's a minor thing, but the way that this will be interpreted, it will result in the needless [creation of an empty text node](https://developer.mozilla.org/en-US/docs/Web/API/Document/createTextNode) when we really don't care for anything to be rendered in the `else` case:

https://codepen.io/aduth/pen/ZoEwNv?editors=1111

What I might suggest instead is:

```jsx
{ !! url && (
	<Toolbar />
) }
```

The boolean coercion is needed because of the way `&&` conditions work in JavaScript, where the left value is effectively _returned_ when falsey. If `url` were an empty string, or worse, a value like 0, it would be falsey but still rendered as a child ([related example](https://codepen.io/aduth/pen/dWwzrL)). React will not render anything for a `false` child.